File: go-sdk/pkg/models/shared/userinvitationpayload.go
package shared

type UserInvitationPayloadLanguageEnum string

const (
	UserInvitationPayloadLanguageEnumEn UserInvitationPayloadLanguageEnum = "en"
	UserInvitationPayloadLanguageEnumDe UserInvitationPayloadLanguageEnum = "de"
)

type UserInvitationPayload struct {
	Email    *string                            `json:"email,omitempty"`
	Language *UserInvitationPayloadLanguageEnum `json:"language,omitempty"`
	Roles    []string                           `json:"roles,omitempty"`
}

File: user-api/ts-sdk/src/sdk/models/shared/userinvitationpayload.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";


export enum UserInvitationPayloadLanguageEnum {
    En = "en",
    De = "de"
}


export class UserInvitationPayload extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=email" })
  email?: string;

  @SpeakeasyMetadata({ data: "json, name=language" })
  language?: UserInvitationPayloadLanguageEnum;

  @SpeakeasyMetadata({ data: "json, name=roles" })
  roles?: string[];
}
